wfl/Models/FoodOrder.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Symphono.Wfl.Models
+{
+    public class FoodOrder
+    {
+        public string Id { get; set; }
```

Comment:
Did we use strings or ObjectId for the Restaurant object?  I'm fine with either, but we need to make sure they're consistent.